rasa/cli/x.py

```diff
@@ -351,6 +353,16 @@ def rasa_x(args: argparse.Namespace) -> None:
 
     _configure_logging(args)
 
+    if version.parse(rasa.version.__version__) >= version.parse(
```

Comment:
I've tried testing this in a fresh venv where ideally I would have first wanted to `pip install` rasa-x, then editable rasa, however the first installation step of rasa-x failed, please see this [thread](https://rasa-hq.slack.com/archives/C36SS4N8M/p1638441134454200). Let me know if there are other ways to test this?

Reply:
Isn't this `if` always going to be true?

Reply:
You can unit test this by monkeypatching versions/constants.
For manual test, it must be possible to install rasa-x 😁 I would just suggest a fresh venv and installing rasa-x first.

Reply:
>  Isn't this if always going to be true?

I was wondering the same thing until i read this comment [here](https://github.com/RasaHQ/rasa/blob/666eac66993682fa907d2aa5fb6f8f984a8f4be6/rasa/version.py#L1) - I assume the version gets overwritten with the one installed by pip? If I misread that comment, how else can I get the user's version?

Reply:
> suggest a fresh venv and installing rasa-x first.

Yeah, I tried that 3 times with fresh venv and the same `psycopg` error came up, Nick also tested on his end and ran in the same error. 
I'll try with monkeypatching in a unit test.

Reply:
Done, ready for review 😄 @joejuzl

Reply:
> I was wondering the same thing until i read this comment here - I assume the version gets overwritten with the one installed by pip? If I misread that comment, how else can I get the user's version?

This version is overridden when we release, and is hardcoded in the code. So if a user installs via pip e.g. `rasa==3.0.1` then the code in the package they install with have `rasa.version.__version__` as `3.0.1`. And that version should always be higher than `MINIMUM_COMPATIBLE_VERSION` in the same code version.

Reply:
Oh I see, thanks for the explanation 🙌🏼

Reply:
So I think we can just check if `rasa.version.__version__` is at least `3.00`